test/extended/threads/atomic_with_offsets.wast

```diff
@@ -0,0 +1,114 @@
+;; atomic operations win non-zero offsets
+
+(module
+  (memory 1 1 shared)
+
+  (func (export "initOffset") (param $value i64) (param $offset i32) (i64.store (local.get $offset) (local.get $value)))
+
+  (func (export "i32.atomic.load") (param $addr i32) (result i32) (i32.atomic.load (local.get $addr)))
```

Comment:
Where is the offset? It should look like:

i32.atomic.load offset=32 (local.get $addr))

The offset does not need to be aligned, just the final address.